### PR TITLE
HSO to OBO

### PR DESCRIPTION
**Pull request by nandadorea**
request from the Health Surveillance Ontology to join the OBO family

### CODE REVIEW

File: ontology/hso.md
Comment (matentzn) [inline ontology/hso.md:3]: seems wrong?

File: ontology/hso.md
Comment (matentzn) [inline ontology/hso.md:29]: Maybe it would be good to give an example of what health surveillance is and also what the term One-health surveillance means.. Its helpful if the description gives at least a sense of what kind of terms the ontology contains.

**Review by matentzn — DISMISSED** [review]

**Review by nandadorea — COMMENTED** [review]
apologies for the error when copying rom another example.

**Comment by jamesaoverton** [comment]
Hi @nandadorea. This looks good. I see one small typo that might as well be fixed now: "is defines as" should be "is defined as". It would also be helpful to add your GitHub username under `contact`:

```
contact:
  email: fernanda.dorea@sva.se
  label: Fernanda Dorea
  github: nandadorea
```

And this branch will need to be updated again before it can be merged.

When making a Pull Request on a fork like this, there's an "Allow edits from maintainers" checkbox. I think it's checked by default, but seems to not be checked for this PR. If that was checked then I could have made these minor edits myself, and saved you some trouble. Here are the docs:

https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by nandadorea** [comment]
I made the changes requested up in this thread, but now I am not sure I did it in the correct place... please advise, and again, apologies, thanks for the patience.

**Review by jamesaoverton — APPROVED** [review]

**Pull request merged**
